obal/data/modules/changelog.py

```diff
@@ -45,7 +45,7 @@ def main():
     user = subprocess.check_output(['rpmdev-packager'], universal_newlines=True).strip()
     evr = get_specfile_evr(spec)
 
-    with open(spec) as spec_file:
+    with open(spec, encoding="utf-8") as spec_file:
```

Comment:
open (encoding=...) is Python 3 only, I wonder why this didn't trip the CI?!

Reply:
Going to disable that for now.